pkg/pillar/cmd/volumemgr/handleconfig.go

```diff
@@ -108,6 +108,15 @@ func vcCreate(ctx *volumemgrContext, objType string, key string,
 		initStatus.LastUse = time.Now()
 		initStatus.PreReboot = false
 		publishVolumeStatus(ctx, initStatus)
+		if !initStatus.DownloadOrigin.HasVerifierRef {
+			if lookupPersistImageStatus(ctx, objType, initStatus.BlobSha256) != nil ||
```

Comment:
Should really have a comment here to say that we can't create a VerifyImageConfig unless the image already exists in the verifier.

Can you add that as part of resolving the conflict?